server/start.go

```diff
@@ -235,27 +234,29 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 
 	app := appCreator(ctx.Logger, db, traceWriter, ctx.Viper)
 
-	nodeKey, err := p2p.LoadOrGenNodeKey(cfg.NodeKeyFile())
-	if err != nil {
-		return err
-	}
-
-	pvFile, err := pvm.LoadOrGenFilePV(cfg.PrivValidatorKeyFile(), cfg.PrivValidatorStateFile())
-	if err != nil {
-		return err
-	}
+	//nodeKey, err := p2p.LoadOrGenNodeKey(cfg.NodeKeyFile())
+	//if err != nil {
+	//	return err
+	//}
+	//
+	//pvFile, err := pvm.LoadOrGenFilePV(cfg.PrivValidatorKeyFile(), cfg.PrivValidatorStateFile())
+	//if err != nil {
+	//	return err
+	//}
 
 	genDocProvider := node.DefaultGenesisDocProviderFunc(cfg)
-	tmNode, err := node.NewNode(
-		cfg,
-		pvFile,
-		nodeKey,
-		proxy.NewLocalClientCreator(app),
-		genDocProvider,
-		node.DefaultDBProvider,
-		node.DefaultMetricsProvider(cfg.Instrumentation),
-		ctx.Logger,
-	)
+	var tmNode node.NodeInterface
+	//tmNode, err = node.NewNode(
+	//	cfg,
+	//	pvFile,
+	//	nodeKey,
+	//	proxy.NewLocalClientCreator(app),
+	//	genDocProvider,
+	//	node.DefaultDBProvider,
+	//	node.DefaultMetricsProvider(cfg.Instrumentation),
+	//	ctx.Logger,
+	//)
+	tmNode, err = optinode.NewNode(proxy.NewLocalClientCreator(app), ctx.Logger)
```

Comment:
Note that this fork of the SDK was mainly created to use lazyledger-core which will still provide a tendermint-consensus node (and not be run with optimint).

On the long run, I'd prefer that any changes necessary to the sdk will be "replayed" via some scaffolding tool that devs use to create their application. For now we certainly need a way to track those changes though. But we won't merge this changes to master: the LL main-chain will still need consensus and hence nodes will run a lazyledger-core (tendermint) node.